velox/expression/tests/ExpressionVerifier.cpp

```diff
@@ -131,25 +150,50 @@ void compareVectors(const VectorPtr& left, const VectorPtr& right) {
   LOG(INFO) << "All results match.";
 }
 
+// Returns a copy of 'rowVector' but with the columns having indices listed in
```

Comment:
Should we actually provide this function as part of VectorFuzzer? Maybe an overload of `VectorFuzzer::wrapInLazyVector()` that takes a vector of column ids?

Reply:
I was thinking the same, but we probably dont want to overload the VectorFuzzer with too many options just yet, if there are similar use cases in other places in the future, we can consolidate those and then add this functionality to the fuzzer. What do you think?